cmd/maya-apiserver/spc-watcher/controller.go

```diff
@@ -128,19 +145,31 @@ func (c *Controller) updateSpc(oldSpc, newSpc interface{}) {
 		// If Resource Version is same it means the object has not got updated.
 		c.queueLoad.Operation = ignoreEvent
 	} else {
+		// Suppressing delete event here as the event is already captured in
+		// deleteSpc hook.
 		if IsDeleteEvent(spcObjectNew) {
-			c.queueLoad.Operation = deleteEvent
+			c.queueLoad.Operation = ignoreEvent
```

Comment:
Please add a log that we are ignoring the delete event here as it will be handled as part of delete operation.

Reply:
Sure